Bio/PDB/Interface.py

```diff
@@ -0,0 +1,226 @@
+
+        sasa_l = [at.xtra['EXP_NACCESS'] for at in structure.get_atoms() if at.get_parent().id[0] == " " and at.element != 'H']
+        sasa = sum(sas_l)
+        return sasa
+        
+    def calculate_BSA(self):
+        "Uses NACCESS module in order to calculate the Buried Surface Area"
+
+        # Extract list of chains in the interface only
+        chains = list(self.get_chains())
+           
+        # Create temporary structures to feed NACCESS
+        structure_A=Structure("chainA")
+        structure_B=Structure("chainB")
+        mA = Model(0)
+        mB = Model(0)
+        mA.add(self.model[chains[0]])
+        mB.add(self.model[chains[1]])
+        structure_A.add(mA)
+        structure_B.add(mB)
+        
+        # Calculate SASAs
+        NACCESS_atomic(self.model)
+        NACCESS_atomic(structure_A[0])
+        NACCESS_atomic(structure_B[0])
+
+        sas_tot= _get_atomic_SASA(self.model)
+        #print 'Accessible surface area, complex:', sas_tot
+        sas_A= _get_atomic_SASA(structure_A)
+        #print 'Accessible surface aream CHAIN A :', sas_A
+        sas_B= _get_atomic_SASA(structure_B)
+        #print 'Accessible surface aream CHAIN B :',sas_B
+        
+        # Calculate BSA
+        bsa = sas_A+sas_B-sas_tot
+                
+        return [bsa, sas_A, sas_B, sas_tot]
+        
+    def _get_atoms_coords(self, interface, opt):
+        "Stores atoms coordinates for both reference and alternate interfaces"
+        
+        ref_atoms = []
+        alt_atoms = []
+        
+        for ref_res, alt_res in zip(self.child_list, interface.child_list):
+            ref_atoms.append(ref_res['CA'])
+            alt_atoms.append(alt_res['CA'])
+            
+        return ref_atoms, alt_atoms
+        
+    def rmsd(self, mobile, mob_ref=None):
+        "Calculates Root Mean Square Deviation between 2 interfaces"
+        
+        if not mob_ref:
+            ref_seq=[to_one_letter_code[r.resname] for r in self.child_list]
+            alt_seq=[to_one_letter_code[r.resname] for r in mobile.child_list]
+            ref_seq.sort()
+            alt_seq.sort()
+            print 'REFERENCE', ref_seq
+            print 'ALTERNATIVE', alt_seq
+            if ref_seq != alt_seq:
+                raise ValueError("Sequences doesn't match")
+        
+        #Build paired lists of c-alpha atoms, ref_atoms and alt_atoms
+        #Possibility to calculate RMSD on CA (ca), backbone (bb) or all atoms (all)        
+        ref_atoms, alt_atoms = self._get_atoms_coords(mobile, opt="ca")
+        
+        #Align these paired atom lists:
+        super_imposer = Superimposer()
+        super_imposer.set_atoms(ref_atoms, alt_atoms)
+        
+        print "RMS = %0.2f" % super_imposer.rms
+        
+        return super_imposer.rms
+        
+    def _get_neighbors_id(self, chain):
+        "Creates a dictionary of tuples from the neighbors dictionary"
+        
+        self.neighbors_id={}
```

Comment:
If it helps, we have a proposed pre-commit hock on issue #493, which would automate checking these little style things.